test/babel/fixtures/class-properties/block-body/expected.js

```diff
@@ -0,0 +1,21 @@
+var _this = this;
+
+class Foo {
+  bar = (...params) => _this.__bar__REACT_HOT_LOADER__(...params);
+
+  __bar__REACT_HOT_LOADER__(a, b) {
+    return a(b);
+  }
+
+}
+;
```

Comment:
not related with this PR I think but what's up with this `;` and the other one at the end?